### PR TITLE
minor formatting for README files

### PR DESCRIPTION
**Pull request by johnkim76**

### CODE REVIEW

File: ec2/README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ These playbooks will:
           $ python -c "import ansible;print(ansible.__version__)"
           2.2.2.0
   * Install python dependencies (This is needed for python2. Use pip2 if using python3)
-     * `pip install boto boto3 six`
+        $ pip install boto boto3 six
```
Comment (jmrodri) [inline ec2/README.md]: This needs to be marked with ``` you can't see it in the formatted view. 

* Install python dependencies (...)
```
$ pip install boto boto3 six
```

something like that

File: ec2/README.md [hunk 2]
```diff
@@ -39,35 +39,32 @@ These playbooks will:
...
-  * `./run_create_infrastructure.sh`
-    * Creates our infrastructure in ec2 if it doesn't exist
-  * `./run_setup_environment.sh`
-    * Sets up OpenShift
-  * In Web Browser
+  * Create our infrastructure in ec2 if it doesn't exist
+        $ ./run_create_infrastructure.sh
+  * Run the setup script
+        $ ./run_setup_environment.sh
+  * Open a Web Browser
```
Comment (jmrodri) [inline ec2/README.md]: You're adding a newline to try to make it easier to read, but it doesn't work. It just ends up being on the same line. I would make all the commands escaped with triple tick marks ```

File: ec2/README.md [hunk 3]
```diff
@@ -39,35 +39,32 @@ These playbooks will:
...
     * Visit: `https://apiserver-service-catalog.USERNAME.ec2.dog8code.com`
-      * Accept the certificate
-      * You will see some text on the screen, ignore this and proceed to the main openshift URL next
-       * Point of this step is just to accept the SSL cert for the apiserver-service-catalog endpoint
-    * Visit: `https://USERNAME.ec2.dog8code.com:8443`
-      * Where username is the value of `whoami` when you launched `run_setup_environment.sh`
+      * Accept the SSL certificate for the apiserver-service-catalog endpoint
+      * Ignore the text that appears and proceed to the main OpenShift URL next
+      * Note: must accept the new SSL cert, each time you reset your OpenShift environment
+    * Visit: `https://<USERNAME>.ec2.dog8code.com:8443`
```
Comment (jmrodri) [inline ec2/README.md]: No changes here.

File: local/README.md [hunk 4]
```diff
@@ -25,30 +23,36 @@ These playbooks will:
...
-To terminate the local instance run the below
-  * `oc cluster down`
+* To terminate the local instance run the below
+      $ oc cluster down
 
-To reset the environment to a clean instance of origin with ASB and Service Catalog run the below
-  * `cd local`
-  * `./reset_environment.sh`
+* To reset the environment to a clean instance of origin with ASB and Service Catalog run the below
+      $ ./reset_environment.sh
```
Comment (jmrodri) [inline local/README.md]: Again, these commands get butted to the end of the sentences and you can't really see they are commands that need executing.

**Comment by johnkim76** [comment]
@jmrodri thank you.  I've made the changes you suggested.

**Pull request merged**